nix/flake/flake.nix

```diff
@@ -0,0 +1,127 @@
+{
+  description = "Docspell flake";
+
+  outputs = { self, nixpkgs }:
+    let
+      supportedSystems = [ "x86_64-linux" "aarch64-linux" ];
+      forAllSystems = nixpkgs.lib.genAttrs supportedSystems;
+      nixpkgsFor = forAllSystems (system: import nixpkgs { inherit system; });
+      # Version config
+      cfg = {
```

Comment:
Taken from the original `pkg.nix`, just for the latest version for brevity's sake.

If the original implementation in `nix/` is kept - the flake could import this attrset from `pkg.nix`